<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotePad</class>
 <widget class="QMainWindow" name="NotePad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1041</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="color">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>More</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bold_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bold_button_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="underline_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="underline_button_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
   </widget>
   <widget class="QPushButton" name="center_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="italic_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="center_button_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="italic_button_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
   </widget>
   <zorder>italic_button_2</zorder>
   <zorder>underline_button_2</zorder>
   <zorder>textEdit</zorder>
   <zorder>fontComboBox</zorder>
   <zorder>comboBox</zorder>
   <zorder>color</zorder>
   <zorder>bold_button</zorder>
   <zorder>bold_button_2</zorder>
   <zorder>underline_button</zorder>
   <zorder>center_button</zorder>
   <zorder>italic_button</zorder>
   <zorder>center_button_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
